HOWTO_RELEASE.md

```diff
@@ -84,12 +80,11 @@ paste, but please pay attention to details!
     deactivate
     cd -
 
-
 ## Release
 
 - release in Github
 
-    xdg-open https://github.com/canonical/craft-cli/tags
+    xdg-open <https://github.com/canonical/craft-cli/tags>
```

Comment:
This is wrong, it will make the command to redirect input/output instead of passing the argument.